tests/test_api.rs

```diff
@@ -11977,3 +11977,24 @@ fn use_counter_callback() {
 
   assert_eq!(COUNT.load(Ordering::Relaxed), 1);
 }
+
+#[test]
+fn test_eternals() {
+  let _setup_guard = setup::parallel_test();
+  let eternal1 = v8::Eternal::empty();
+
+  let mut isolate = v8::Isolate::new(Default::default());
+  let mut scope = v8::HandleScope::new(&mut isolate);
+
+  let str1 = v8::String::new(&mut scope, "hello").unwrap();
+
+  assert!(eternal1.is_empty());
+  eternal1.set(&mut scope, str1);
+  assert!(!eternal1.is_empty());
+
+  let str1_get = eternal1.get(&mut scope);
+  assert_eq!(str1, str1_get);
+
+  eternal1.clear();
+  assert!(eternal1.is_empty());
+}
```

Comment:
Is there a way to test behavior after the isolate is dropped? Ie. it won't segfault when accessed

Reply:
Yep, added.